lib/src/intl_suggestors/intl_message_syntax.dart

```diff
@@ -54,8 +54,13 @@ class MessageSyntax {
   String functionNameFor(
     StringInterpolation node,
     String namePrefix,
-  ) =>
-      '${getTestId(null, node) ?? '${namePrefix}_intlFunction'}';
+  ) {
+    var ignoreNewLineString = node.toSource();
```

Comment:
Actually, here our `single line with an explicit new line` test case is failing [here](https://github.com/Workiva/over_react_codemod/blob/master/test/intl_suggestors/utils_test.dart#L104)  that's why we added a condition if we have a new line inside the string so it will ignore it and create function name with intlFuction0